src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/Summary.tsx

```diff
@@ -54,6 +54,11 @@ export function Summary({
         {text}
       </ThemeText>
     );
+    const requiredOnMobile = fareProductTypeConfig.configuration
+      .requiresTokenOnMobile
+      ? summary(t(PurchaseOverviewTexts.summary.messageRequiresMobile))
+      : null;
```

Comment:
requiredOnMobile sounds like a boolean. Would suggest renaming to requiredOnMobileText or just inlining directly in jsx below like `{fareProductTypeConfig.configuration.requiresTokenOnMobile && summary(t(PurchaseOverviewTexts.summary.messageRequiresMobile))}`